apps/common/utils.py

```diff
@@ -16,13 +16,15 @@
 )
 
 
-def get_login(data):
-    return f"{(data[:4])}-{randint(100, 999)}".lower()
+def get_generated_username(last_name: str, first_name: str) -> str:
+    if len(last_name) < 4:
+        last_name = last_name.join(first_name)
+    transliterated_last_name = translit(last_name)
+    return get_login(transliterated_last_name)
 
 
-def get_generated_username(full_name):
-    transliterated_full_name = translit(full_name)
-    return get_login(transliterated_full_name)
+def get_login(data: str) -> str:
```

Comment:
Are you sure about this function name?
Maybe we can find another better understanding name for it.

Reply:
of course no, bcs i asked Victor to change it, but we didn't find any good name for this function. i'll change it, ok

Reply:
@victor-nuzhniy please rename this method